packages/design-system/src/components/toggle-group.tsx

```diff
@@ -30,11 +31,12 @@ export const Item = styled(ToggleGroupPrimitive.Item, {
   },
   "&:last-child": { borderTopRightRadius: 4, borderBottomRightRadius: 4 },
   "&:hover": { backgroundColor: "$slateA3" },
+  "&:focus": { boxShadow: "0 0 0 $spacing$2 $colors$blue10", zIndex: 1 },
   "&[data-state=on]": {
-    backgroundColor: "$slateA5",
-  },
-  "&:focus": {
-    position: "relative",
-    boxShadow: `0 0 0 2px black`,
+    backgroundColor: "$slate5",
+    "&[data-breakpoint=true]": {
```

Comment:
I feel like we talked about it several times by now.  A design system component doesn't know the designer specific stuff like breakpoint. It's business logic related